extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_logo.php

```diff
@@ -0,0 +1,18 @@
+<? if ( !isset( $model['header']['subtitle'] ) ): ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)